miniupnpc/CMakeLists.txt

```diff
@@ -56,40 +56,24 @@ configure_file (${CMAKE_CURRENT_SOURCE_DIR}/miniupnpcstrings.h.cmake ${CMAKE_CUR
 target_include_directories(miniupnpc-private INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>)
 
 set (MINIUPNPC_SOURCES
-  igd_desc_parse.c
-  miniupnpc.c
-  minixml.c
-  minisoap.c
-  minissdpc.c
```

Comment:
Because `minissdpc.c` has been removed here, I have the following error on Windows (MSVC) with CMake:
```
libminiupnpc.lib(miniupnpc.c.obj) : error LNK2019: unresolved external symbol ssdpDiscoverDevices referenced in function upnpDiscover
```

Reply:
that should be fixed in 2017ae036f67f1a47b49191b72a6ac9b262fda3e

Reply:
I confirmed, it works now, thank you.